### ISSUE TITLE
Abnormal circulating Complement component C1q receptor concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5993]
**Preferred term label:** Abnormal circulating Complement component C1q receptor concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of Complement component C1q receptor in the blood is outside the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Complement component C1q receptor concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of Complement component C1q receptor in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Complement component C1q receptor concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Complement component C1q receptor concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of Complement component C1q receptor in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Complement component C1q receptor concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is to my knowledge not a test that has well defined normal ranges so that we could say whether a given value is abnormal or not.
